salon_creator/lib/widgets/custom_textfield.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:salon_creator/widgets/custom_label.dart';
+
+class TextFieldWithLabel extends StatefulWidget {
+  const TextFieldWithLabel({
+    Key key,
+    this.controller,
+    this.height,
+    this.width,
+    this.keyboardType,
+    this.title,
+    this.labelWidth,
+    this.labelHeight,
+    this.hintText,
+  }) : super(key: key);
+  final TextEditingController controller;
+  final double height;
+  final double width;
+  final TextInputType keyboardType;
+  final String title;
+  final double labelWidth;
+  final double labelHeight;
+  final String hintText;
+
+  @override
+  _TextFieldWithLabelState createState() => _TextFieldWithLabelState();
+}
+
+class _TextFieldWithLabelState extends State<TextFieldWithLabel> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: widget.height,
+      width: widget.width,
+      child: Column(
+        children: [
+          CustomLabel(
+            title: widget.title,
+            width: widget.labelWidth,
+            height: widget.labelHeight,
+          ),
+          SizedBox(
+            height: 12,
+          ),
+          _buildCustomTextField(
+            widget.controller,
```

Comment:
No need to pass these parameters here, since they are available at global scope within the **_TextFieldWithLabelState** class.